.idea/codeStyles/Project.xml

```diff
@@ -1,17 +1,11 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
     <option name="LINE_SEPARATOR" value="&#10;" />
-    <AndroidXmlCodeStyleSettings>
-      <option name="USE_CUSTOM_SETTINGS" value="true" />
-    </AndroidXmlCodeStyleSettings>
     <JavaCodeStyleSettings>
       <option name="FIELD_NAME_PREFIX" value="m" />
       <option name="STATIC_FIELD_NAME_PREFIX" value="s" />
       <option name="CLASS_COUNT_TO_USE_IMPORT_ON_DEMAND" value="99999" />
       <option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="99999" />
-      <option name="PACKAGES_TO_USE_IMPORT_ON_DEMAND">
```

Comment:
Why removing this?

Reply:
`3.5` changed the codestyle settings. This is automatic done by ide